### ISSUE TITLE
Something went wrong with the epoch number

### ISSUE THREAD

**Issue by g3mini28** [davidsandberg/facenet#994]
Hi,

I am currently trying to finetune the VGGFace2 pretrained model and train using triplet loss. I am training on my own dataset with the following parameters:

python src/train_tripletloss.py 
--logs_base_dir logs/facenet/TripletLoss_Training
--models_base_dir models/facenet/tripletloss 
--data_dir dataset/own_dataset_160 
--image_size 160 --model_def models.inception_resnet_v1 
--optimizer RMSPROP 
--learning_rate 0.01 
--weight_decay 1e-4 
--pretrained_model Pre-Trained_Model/Inception_ResNet_v1/VGGFace2/20180402-114759/model-20180402-114759.ckpt-275 
--embedding_size 512 
--images_per_person 10 
--people_per_batch 6 
--batch_size 12 
--epoch_size 100 
--max_nrof_epoch 5

I have 10 classes with 10 images each, so a total of 100 images. However, the epoch number jumped from 3 to 5, and after 100 iterations, it did not refresh. Instead continuing till 130+, what went wrong and is there something which I have done wrong? Any help would be much appreciated. 

<img width="314" alt="Capture" src="https://user-images.githubusercontent.com/35353866/54708037-6cced400-4b7d-11e9-96dc-508cfbc3a07f.PNG">

**Comment by thelastfunction** [comment]
for finetuning 10 classes are very very less...you should try fine tuning with more data (around 1000+ classes)
